### PR TITLE
Crncc 1717

### PR DESCRIPTION
**Pull request by ThomasJackSmolen**
In the ManualEntry.tsx file
 - removed the (optional) part of the postcode label
 - added a required field in the rules with an error message to alert user to enter a postcode

### CODE REVIEW

**Review by paddyduff31 — CHANGES_REQUESTED** [review]
- Remove unnecessary comments

File: src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx [hunk 1]
```diff
@@ -94,8 +104,15 @@ describe("ManualEntry", () => {
...
-      expect(screen.getByText("Enter the first line of your address")).toBeInTheDocument();
-      expect(screen.getByText("Enter the town of your address")).toBeInTheDocument();
+
+      expect(
+        screen.getByText("Enter the first line of your address")
+      ).toBeInTheDocument();
+      expect(
+        screen.getByText("Enter the town of your address")
+      ).toBeInTheDocument();
+      expect(screen.getByText("Enter a real postcode")).toBeInTheDocument();
```
Comment (kevinofficer) [inline src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx:114]: Your test expects "enter a real postcode", but your component message renders "Enter your postcode", update one of these to remain consistent with the rest of the application.

File: src/components/Shared/FormElements/AddressFormComponents/ManualEntry.tsx [hunk 2]
```diff
@@ -170,6 +170,7 @@ function ManualEntry(props: ManualEntryProps) {
             />
           )}
           rules={{
+            required: "Enter your postcode",
```
Comment (kevinofficer) [inline src/components/Shared/FormElements/AddressFormComponents/ManualEntry.tsx:173]: update is necessary to match the test above

File: src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx [hunk 3]
```diff
@@ -81,8 +86,13 @@ describe("ManualEntry", () => {
...
 
-      expect(screen.getByText("Enter the first line of your address")).toBeInTheDocument();
-      expect(screen.getByText("Enter the town of your address")).toBeInTheDocument();
+      expect(
+        screen.getByText("Enter the first line of your address")
+      ).toBeInTheDocument();
+      expect(
+        screen.getByText("Enter the town of your address")
+      ).toBeInTheDocument();
+      expect(screen.getByText("Enter your postcode")).toBeInTheDocument();
```
Comment (kevinofficer) [inline src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx:95]: is "enter your postcode" the correct validation message?

File: src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx [hunk 4]
```diff
@@ -65,8 +65,13 @@ describe("ManualEntry", () => {
         target: { value: "test town" },
       });
 
+      fireEvent.input(screen.getByLabelText("Postcode"), {
+        target: { value: "test postcode" },
+      });
+
       expect(screen.getByDisplayValue("test address 1")).toBeInTheDocument();
       expect(screen.getByDisplayValue("test town")).toBeInTheDocument();
+      expect(screen.getByDisplayValue("test postcode")).toBeInTheDocument();
```
Comment (kevinofficer) [inline src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx:74]: now that a real postcode is required, is this causing a failure? use a real postcode format 
e.g. PA London, SW1E 5DN

File: src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx [hunk 5]
```diff
@@ -65,8 +65,13 @@ describe("ManualEntry", () => {
         target: { value: "test town" },
       });
 
+      fireEvent.input(screen.getByLabelText("Postcode"), {
+        target: { value: "test postcode" },
```
Comment (kevinofficer) [inline src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx:69]: now that a real postcode is required, is this causing a failure? use a real postcode format 
e.g. PA London, SW1E 5DN

**Review by kevinofficer — CHANGES_REQUESTED** [review]

File: src/components/Shared/FormElements/AddressFormComponents/ManualEntry.tsx [hunk 6]
```diff
@@ -172,6 +172,9 @@ function ManualEntry(props: ManualEntryProps) {
           rules={{
             validate: (value) => {
               const cleaned = value.replace(/[^A-Za-z0-9]/g, "");
+              if (value.trim() === "") {
```
Comment (paddyduff31) [inline src/components/Shared/FormElements/AddressFormComponents/ManualEntry.tsx:175]: Can we change this back to the original implementation please?

**Review by paddyduff31 — APPROVED** [review]

**Pull request merged**